docs/data/toolpad/core/pages.ts

```diff
@@ -81,28 +82,23 @@ const pages: MuiPage[] = [
           },
         ],
       },
-    ],
-  },
-  {
-    pathname: '/toolpad/core/api-group',
-    title: 'APIs',
-    children: [
       {
-        pathname: '/toolpad/core/api/components-group',
-        subheader: 'Components',
+        pathname: '/toolpad/core/components/data-group',
+        subheader: 'Data',
         children: [
           {
-            pathname: '/toolpad/core/api/app-provider',
-            title: 'AppProvider',
-          },
-          {
-            pathname: '/toolpad/core/api/dashboard-layout',
-            title: 'DashboardLayout',
+            pathname: '/toolpad/core/components/data-grid',
```

Comment:
I changed these component pages to be like `toolpad/core/components/data-grid` in the PR for generating API docs, just to follow the convention I saw in other MUI libraries. But I wasn't sure about it, so we can have the like this and remove the `react-` part.